geode-for-redis/src/main/java/org/apache/geode/redis/internal/data/RedisList.java

```diff
@@ -287,15 +287,15 @@ public Void ltrim(int start, int end, Region<RedisKey, RedisData> region,
     return null;
   }
 
-  private int getBoundedStartIndex(long index, int size) {
+  private int getBoundedStartIndex(int index, int size) {
     if (index >= 0L) {
       return (int) Math.min(index, size);
     } else {
       return (int) Math.max(index + size, 0);
```

Comment:
These casts are now redundant and can be removed, as are the ones in `getBoundedEndIndex()`.

Reply:
Swept away.